src/main/java/com/Novi/TechItEasy/models/CI_Module.java

```diff
@@ -0,0 +1,67 @@
+package com.Novi.TechItEasy.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
+
+import java.util.List;
+
+@Entity
+@Table(name = "CI_Modules")
+public class CI_Module {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false, updatable = false)
+    private Long id;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "type")
+    private String type;
+
+    @Column(name = "price")
+    private Double price;
+
+    @JsonIgnore
```

Comment:
Je zou geen JSON annotaties moeten hoeven gebruiken in je Entities. Je entities zouden niet voorbij de Service-laag moeten komen en dus ook nooit tot JSON hoeven worden geserialised.

Reply:
Dat gezegd hebbende, is dit wel een handige annotatie om bijvoorbeeld recursie of endless loops tegen te gaan bij JSON serialization